### PR TITLE
Add solution architecture diagram

### PR DESCRIPTION
**Pull request by aaron-schnieder**
Added solution architecture diagram
Added alt-text to k8 resources diagram

### CODE REVIEW

File: Deployment.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ The following resources are deployed by [deployCluster.sh](deployment/deployClus
 - Azure Traffic Manager endpoint to cluster
 
 The following diagram shows the **physical artifacts** created in the resource group.
-![](./deployment/kubernetes-resource-group.png)
+![Kubernetes Resources](./deployment/kubernetes-resource-group.png)
```
Comment (billpratt) [inline Deployment.md]: "etcd" mispelled in image
image

Comment (billpratt) [reply Deployment.md]: Also, I would change the "docker" layer to "kubelet".

Comment (billpratt) [reply Deployment.md]: We could mimic this diagram for the Kube setup bits. I'd leave off cAdvisor

https://upload.wikimedia.org/wikipedia/commons/b/be/Kubernetes.png

Comment (aaron-schnieder) [reply Deployment.md]: @jobanputra created the diagram above in a separate a while back. @billpratt could you open a bug in VSTS for those items?

Comment (billpratt) [reply Deployment.md]: Will do

Comment (jobanputra) [reply Deployment.md]: I created another pull request with changes.

**Pull request merged**